packages/@sanity/default-layout/src/sideMenu/SideMenu.tsx

```diff
@@ -2,90 +2,136 @@
 ///<reference types="@sanity/types/parts" />
 
 import {UserAvatar, useZIndex} from '@sanity/base/components'
-import {Layer} from '@sanity/ui'
-import React from 'react'
-import CloseIcon from 'part:@sanity/base/close-icon'
-import SignOutIcon from 'part:@sanity/base/sign-out-icon'
-import Button from 'part:@sanity/components/buttons/default'
+import {Layer, Card, Flex, Text, Box, Button, Stack, useGlobalKeyDown} from '@sanity/ui'
+import {CloseIcon, LeaveIcon} from '@sanity/icons'
+import React, {useEffect, useRef} from 'react'
 import ToolMenu from 'part:@sanity/default-layout/tool-switcher'
-import {DatasetSelect} from '../components'
-import {Router, Tool, User} from '../types'
+import styled from 'styled-components'
+import {DatasetSelect} from '../datasetSelect'
+import {Tool, User} from '../types'
 import {HAS_SPACES} from '../util/spaces'
 
-import styles from './SideMenu.css'
-
 interface Props {
   activeToolName: string | null
   isOpen: boolean
   onClose: () => void
   onSignOut: () => void
   onSwitchTool: () => void
-  router: Router
   tools: Tool[]
   user: User
 }
 
-function SideMenu(props: Props) {
-  const {activeToolName, isOpen, onClose, onSignOut, onSwitchTool, router, tools, user} = props
-  const zindex = useZIndex()
-  let className = styles.root
-  if (isOpen) className += ` ${styles.isOpen}`
+const Root = styled(Layer)`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  pointer-events: none;
+`
+
+const Backdrop = styled(Box)<{$open: boolean}>`
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  opacity: ${({$open}) => ($open ? 1 : 0)};
+  background: var(--card-shadow-penumbra-color);
+  transition: 200ms opacity ease-in-out;
+  pointer-events: ${({$open}) => ($open ? 'all' : 'none')};
+`
+
+const InnerCard = styled(Card)<{$open: boolean}>`
+  position: relative;
+  pointer-events: all;
+  flex-direction: column;
+  min-width: 200px;
+  max-width: 280px;
+  transform: ${({$open}) => ($open ? 'translate(0%)' : 'translate(-100%)')};
```

Comment:
This change will make sure the 1px shadow border is not visible, and that `translate3d` enables hardware acceleration:

```suggestion
  transform: ${({$open}) =>
    $open ? 'translate3d(0, 0, 0)' : 'translate3d(calc(-100% - 1px), 0, 0)'};
```